README.md

```diff
@@ -31,8 +31,9 @@
 
 <div align="center">
 
-<img src="./Images/Hsoc22Logo.jpg" width="20%" height="20%">
-  
+ <img src="./Images/Hsoc22Logo.jpg" width="20%" height="20%">
+ 
+ <img src="./Images/ssoc22.jpg" width="20%" height="40%">  
```

Comment:
can' t the height of the both images  be same and also increase the space b/w them

Reply:
yep was thinking of it but the css wasnt getting applied to ssoc logo , let me look once more :eye:

Reply:
@legendarykamal  any updates